core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanNodeStatsSummarizer.java

```diff
@@ -202,6 +203,7 @@ private static List<PlanNodeStats> getPlanNodeStats(TaskStats taskStats)
                         new Duration(planNodeBlockedMillis.get(planNodeId), MILLISECONDS),
                         planNodeInputPositions.get(planNodeId),
                         succinctBytes(planNodeInputBytes.get(planNodeId)),
+                        succinctBytes(planNodePhysicalInputDataSize.getOrDefault(planNodeId, 0L)),
```

Comment:
Do we need to add it here ? Physical input data size is only about source nodes AFAIK